packages/data-helper/src/hooks/useSchemaFields/useSchemaFields.jsx

```diff
@@ -0,0 +1,7 @@
+import {useQuery} from 'react-apollo';
+import {SCHEMA_FIELDS_QUERY} from './useSchemaFields.gql-queries';
+
+export const useSchemaFields = variables => {
+    const {data} = useQuery(SCHEMA_FIELDS_QUERY, {variables: variables});
```

Comment:
How is loading handled ? You must return the loading state and handle it in the called (useNodeInfo). If it needs the schema it should wait for it to be loaded and returns also a  loading state